test/ngAria/ariaSpec.js

```diff
@@ -526,6 +538,31 @@ describe('$aria', function() {
       element.triggerHandler({ type: 'keypress', keyCode: 13 });
       expect(element.text()).toBe('keypress13');
     });
+
+    it('should not bind keypress to non-div elements', function() {
```

Comment:
Might be good to note in the spec title that this is a symptom of using the default config. (This might be spelled out clearly higher up, can't tell on mobile view)